.github/workflows/cfn-publish-resource.yaml

```diff
@@ -59,24 +59,24 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY_PUBLISHING}}
       AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID_PUBLISHING}}
 
-      # ldap-configuration
-      LDAP_BIND_PASSWORD: ${{ secrets.LDAP_BIND_PASSWORD }}
-      LDAP_HOST_NAME: ${{ secrets.LDAP_HOST_NAME }}
-      LDAP_BIND_USER_NAME: ${{ secrets.LDAP_BIND_USER_NAME }}
+      # LDAP config
+      LDAP_HOST_NAME: ${{ secrets.LDAP_HOST_NAME_PUBLISHING }}
+      LDAP_BIND_USER_NAME: ${{ secrets.LDAP_BIND_USER_NAME_PUBLISHING }}
+      LDAP_BIND_PASSWORD: ${{ secrets.LDAP_BIND_PASSWORD_PUBLISHING }}
 
       # third party integration
-      WEBHOOK_CREATE_URL: ${{ secrets.WEBHOOK_CREATE_URL }}
-      WEBHOOK_UPDATE_URL: ${{ secrets.WEBHOOK_UPDATE_URL }}
-      WEBHOOK_UPDATE_SECRET: ${{ secrets.WEBHOOK_UPDATE_SECRET }}
-      PROMETHEUS_USER_NAME: ${{ secrets.PROMETHEUS_USER_NAME }}
-      PROMETHEUS_PASSWORD_NAME: ${{ secrets.PROMETHEUS_PASSWORD_NAME }}
-      PAGER_DUTY_CREATE_SERVICE_KEY: ${{ secrets.PAGER_DUTY_CREATE_SERVICE_KEY }}
-      PAGER_DUTY_UPDATE_SERVICE_KEY: ${{ secrets.PAGER_DUTY_UPDATE_SERVICE_KEY }}
-      DATA_DOG_CREATE_API_KEY: ${{ secrets.DATA_DOG_CREATE_API_KEY }}
-      DATA_DOG_UPDATE_API_KEY: ${{ secrets.DATA_DOG_UPDATE_API_KEY }}
-      OPS_GENIE_API_KEY: ${{ secrets.OPS_GENIE_API_KEY }}
-      MICROSOFT_TEAMS_WEBHOOK_CREATE_URL: ${{ secrets.MICROSOFT_TEAMS_WEBHOOK_CREATE_URL }}
-      MICROSOFT_TEAMS_WEBHOOK_UPDATE_URL: ${{ secrets.MICROSOFT_TEAMS_WEBHOOK_UPDATE_URL }}
+      WEBHOOK_CREATE_URL: ${{ secrets.WEBHOOK_CREATE_URL_PUBLISHING }}
+      WEBHOOK_UPDATE_URL: ${{ secrets.WEBHOOK_UPDATE_URL_PUBLISHING }}
+      WEBHOOK_UPDATE_SECRET: ${{ secrets.WEBHOOK_UPDATE_SECRET_PUBLISHING }}
+      PROMETHEUS_USER_NAME: ${{ secrets.PROMETHEUS_USER_NAME_PUBLISHING }}
+      PROMETHEUS_PASSWORD_NAME: ${{ secrets.PROMETHEUS_PASSWORD_NAME_PUBLISHING }}
+      PAGER_DUTY_CREATE_SERVICE_KEY: ${{ secrets.PAGER_DUTY_CREATE_SERVICE_KEY_PUBLISHING }}
+      PAGER_DUTY_UPDATE_SERVICE_KEY: ${{ secrets.PAGER_DUTY_UPDATE_SERVICE_KEY_PUBLISHING }}
+      DATA_DOG_CREATE_API_KEY: ${{ secrets.DATA_DOG_CREATE_API_KEY_PUBLISHING }}
+      DATA_DOG_UPDATE_API_KEY: ${{ secrets.DATA_DOG_UPDATE_API_KEY_PUBLISHING }}
+      OPS_GENIE_API_KEY: ${{ secrets.OPS_GENIE_API_KEY_PUBLISHING }}
+      MICROSOFT_TEAMS_WEBHOOK_CREATE_URL: ${{ secrets.MICROSOFT_TEAMS_WEBHOOK_CREATE_URL_PUBLISHING }}
+      MICROSOFT_TEAMS_WEBHOOK_UPDATE_URL: ${{ secrets.MICROSOFT_TEAMS_WEBHOOK_UPDATE_URL_PUBLISHING }}
```

Comment:
why did you add _PUBLISHING at the end?

Reply:
I look at the repo secrets and this is the name of the secrets as they were created

Reply:
they've used only for some resources.  It works fine for resource that don't need these variables, it'll mainly fail for ldap or third-party resources.

Reply:
even if they did not use the github action, the publishing.sh script is used also by the old release process. With this change, the old release process will fail. I would update the secrets in the action and repo instead so we can still use the old release process if needed.

Reply:
sorry, nvm my message, I see that you indeed added the secret to the repo. I got confused 😕